caom2repo/README.rst

```diff
@@ -1,10 +1,6 @@
 =========
 Caom2Repo
 =========
-
-.. image:: https://img.shields.io/pypi/v/caom2repo.svg   
-    :target: https://pypi.python.org/pypi/caom2repo
-
 Client caom2-repo
```

Comment:
Please add this back. It displays an icon with the version number published on pypi.